ui/src/organizations/components/GetOrgResources.tsx

```diff
@@ -97,6 +97,10 @@ export default class GetOrgResources<T> extends PureComponent<
     resources: T[],
     key: string | number | symbol
   ): key is keyof T {
-    return key in resources[0]
+    const resource = resources[0]
```

Comment:
maybe also `resource = get(resources, '0', null)` ? just in case resources comes back undefined or not an array?